src/js/components/search/table/ResultsColumnVisibleOption.jsx

```diff
@@ -94,7 +94,9 @@ class ResultsColumnVisibleOption extends React.Component {
 
     render() {
         const checked = this.props.checked;
-        const { label, isDragging, connectDragSource, connectDropTarget } = this.props;
+        const {
```

Comment:
I don't think we're using this pattern anywhere else

Reply:
This is pretty great.